core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -74,14 +79,18 @@ public void migrate() {
       if (deployResourceCommandStep2 != null) {
         deployResourceCommandStep2.send().join();
       }
+      LOGGER.debug("Completed auto deployment.");
     }
 
     String latestLegacyId = idKeyMapper.findLatestIdByType("runtimeProcessInstance");
+    LOGGER.debug("Latest legacy id is {}", latestLegacyId);
```

Comment:
🙃 Maybe we could be a bit more precise here:
```suggestion
    LOGGER.debug("Legacy ID of latest migrated process instance: [{}]", latestLegacyId);
```